Client/Frontend/Settings/FxAContentViewController.swift

```diff
@@ -118,6 +123,21 @@ class FxAContentViewController: SettingsContentViewController, WKScriptMessageHa
         helper.application(app, didReceiveAccountJSON: data)
     }
 
+    @objc fileprivate func SELdidVerify(_ notification: Notification) {
```

Comment:
We've stopped using the `SEL` prefix I think. Does it make sense here to remove it?